test/helper/storage.test.js

```diff
@@ -60,7 +60,7 @@ describe('helpers storage', function() {
       var nothing = storage.getItem('data');
       expect(nothing).to.be(undefined);
     });
-    it('should call getItem and return undefined when value is undefined', function() {
+    it('should call getItem and return undefined when value is null', function() {
```

Comment:
again this is wrong. if it "should return undefined when value is null" this is not happening. The logic tested here is returning null.

Reply:
ahh yeah.. fixed